### PR TITLE
Deploy new contracts on Testnet

### PR DESCRIPTION
**Pull request by mzxyz**
## Changes

- Deploy root and child contracts

## Todo

- [x] Add the `ChildERC20` address once Polygon team compete the token mapping
- [ ] Contract call for child contract settings (TX failed).
- [x] Redeploy `ProjectRegistry` contract
- [x] Testnet Setup
   - [x] Set inflation destination & xcRecipient
   - [x] SQToken deposit
   - [x] Create projects
   - [x] Create planTemplates
   - [x] Token Exchange order
   - [x] Allocate token to indexers

### CODE REVIEW

File: scripts/contracts.ts [hunk 1]
```diff
@@ -102,6 +104,7 @@ export type Contracts = {
     consumerRegistry: ConsumerRegistry;
     priceOracle: PriceOracle;
     polygonDestination: PolygonDestination;
+    rootChainMananger: RootChainManager;
```
Comment (ianhe8x) [inline scripts/contracts.ts]: Since we don't need to deploy root chain manager, we can leave it out from here.

File: scripts/contracts.ts [hunk 2]
```diff
@@ -160,6 +163,7 @@ export const CONTRACT_FACTORY: Record<ContractName, FactoryContstructor> = {
     PriceOracle: PriceOracle__factory,
     ChildERC20: ChildERC20__factory,
     PolygonDestination: PolygonDestination__factory,
+    RootChainManager: RootChainManager__factory,
```
Comment (ianhe8x) [inline scripts/contracts.ts]: Since we don't need to deploy root chain manager, we can leave it out from here.

File: scripts/contracts.ts (hunk 1 above)
Comment (mzxyz) [reply scripts/contracts.ts]: Do we still have scenarios to consume it through contract SDK?

Comment (ianhe8x) [reply scripts/contracts.ts]: > Do we still have scenarios to consume it through contract SDK?

I have used `@maticnetwork/maticjs` so we don't need to directly invoke root chain manager

**Pull request merged**